Makefile

```diff
@@ -73,13 +77,36 @@ osx-package: $(OSX_FONT)
 	cp README.md $(OSX_PACKAGE)
 	7z a -tzip -mx=9 $(OSX_PACKAGE).zip ./$(OSX_PACKAGE)
 
+windows-package: $(WINDOWS_FONT)
+	cp $(WINDOWS_FONT) .
+
 # Build both versions of the fonts
 $(REGULAR_FONT): $(SVG_BW_FILES) $(SVG_COLOR_FILES) copy-extra
 	$(SCFBUILD) -c scfbuild.yml -o $(REGULAR_FONT) --font-version="$(VERSION)"
 
 $(OSX_FONT): $(SVG_BW_FILES) $(SVG_COLOR_FILES) copy-extra
 	$(SCFBUILD) -c scfbuild-osx.yml -o $(OSX_FONT) --font-version="$(VERSION)"
 
+$(WINDOWS_FONT): $(TMP)/svg.ttx $(TMP)/name.ttx $(REGULAR_FONT)
+	# merge segoe ui symbol into emoji one, this keeps emoji one's glyph ids intact
+	# for the 'SVG ' table data
+	pyftmerge $(REGULAR_FONT) $(SEGOE_UI_FONT)
```

Comment:
Hmm... Not on my machine. I have ttx, but not pyftmerge. Maybe it's not in the Ubuntu package. How did you install fonttools?